ament_tools/build_type.py

```diff
@@ -53,7 +53,7 @@ class BuildAction:
     the optional ``env`` parameter.
     """
 
-    def __init__(self, cmd, type='command', title=None, dry_run_cmd=None,
+    def __init__(self, cmd, type='command', title=None, dry_run_cmd=None,  # noqa
```

Comment:
could you point out what the error we turn off here ? Would another wrapping strategy allow us to pass the test without ignoring the error?

Reply:
`type` is a built-in and shouldn't be used as a variable name.